components/brave_policy/ad_block_only_mode/ad_block_only_mode_policies_utils.cc

```diff
@@ -0,0 +1,55 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_policy/ad_block_only_mode/ad_block_only_mode_policies_utils.h"
+
+#include "base/values.h"
+#include "build/build_config.h"
+#include "components/content_settings/core/common/content_settings.h"
+#include "components/policy/core/common/policy_bundle.h"
+#include "components/policy/core/common/policy_map.h"
+#include "components/policy/core/common/policy_namespace.h"
+#include "components/policy/core/common/policy_types.h"
+#include "components/policy/policy_constants.h"
+
+void LoadAdBlockOnlyModePolicies(policy::PolicyBundle& bundle) {
```

Comment:
why is this in a separate utils file?

Reply:
and this belongs in components/brave_shields just like the brave origin stuff is in components/brave_origin